cpp/include/nvtx3/nvtx3.hpp

```diff
@@ -2131,6 +2176,7 @@ inline void mark(event_attributes const& attr) noexcept
 
 }  // namespace nvtx3
 
+#ifndef NVTX_DISABLE
```

Comment:
Is this intentional?

Reply:
Yes, it's a bit hard to see with the diff but I just moved the comment for `NVTX3_V1_FUNC_RANGE_IN` below the `#ifdef NVTX_DISABLE` statement.  I feel the comment should be directly on top of the function definition and not separated by the preprocessor condition.

Reply:
Agreed